kyuubi-hive-jdbc/src/main/java/org/apache/kyuubi/jdbc/hive/KyuubiConnection.java

```diff
@@ -78,6 +78,7 @@
 public class KyuubiConnection implements java.sql.Connection, KyuubiLoggable {
   public static final Logger LOG = LoggerFactory.getLogger(KyuubiConnection.class.getName());
   public static final String BEELINE_MODE_PROPERTY = "BEELINE_MODE";
+  public static final int DEFAULT_ENGINE_LOG_THREAD_TIMEOUT = 10 * 1000;
```

Comment:
Could this add an option for log thread timeout?

Reply:
it is not needed, just a thread timeout.

Reply:
Just refer: https://github.com/apache/hive/blob/2d05298f5b4a40b5eb339efcd0701e31b694ccc7/beeline/src/java/org/apache/hive/beeline/Commands.java#L75

Reply:
the thread.join should return very fast, the timeout is for the worst case.

Reply:
@turboFei, IMO, the timeout value is configurable, whatever any reason, which could be 5*1000 etc.

Reply:
It should not a big deal, we can remove the `final`, if users really want to change it, they can re-assign the value of the static variable.

Reply:
@pan3793, I agree with you for the point that removes the `final`.

Reply:
thanks, resolved.